projects/ad9656/zcu102/system_constr.xdc

```diff
@@ -0,0 +1,38 @@
+set_property  -dict {PACKAGE_PIN  F1} [get_ports rx_data_n[1]]                                        ; ## A07  FMC_HPC0_DP2_M2C_N
+set_property  -dict {PACKAGE_PIN  H2} [get_ports rx_data_p[2]]                                        ; ## C06  FMC_HPC0_DP0_M2C_P
+set_property  -dict {PACKAGE_PIN  H1} [get_ports rx_data_n[2]]                                        ; ## C07  FMC_HPC0_DP0_M2C_N
+set_property  -dict {PACKAGE_PIN  K2} [get_ports rx_data_p[3]]                                        ; ## A10  FMC_HPC0_DP3_M2C_P
+set_property  -dict {PACKAGE_PIN  K1} [get_ports rx_data_n[3]]                                        ; ## A11  FMC_HPC0_DP3_M2C_N
+set_property  -dict {PACKAGE_PIN  Y2  IOSTANDARD LVCMOS18} [get_ports spi_csn_ad9508]                 ; ## G09  FMC_HPC0_LA03_P
+set_property  -dict {PACKAGE_PIN  Y1  IOSTANDARD LVCMOS18} [get_ports spi_csn_ad9553]                 ; ## G10  FMC_HPC0_LA03_N
+#set_property  -dict {PACKAGE_PIN  Y4  IOSTANDARD LVCMOS18} [get_ports ref_clk0_p]                     ; ## G06  FMC_HPC0_LA00_CC_P
+#set_property  -dict {PACKAGE_PIN  Y3  IOSTANDARD LVCMOS18} [get_ports ref_clk0_n]                     ; ## G07  FMC_HPC0_LA00_CC_N
+set_property  -dict {PACKAGE_PIN  AB4  IOSTANDARD LVCMOS18 } [get_ports spi_clk]                          ; ## D08  FMC_HPC0_LA01_CC_P
+set_property  -dict {PACKAGE_PIN  AC4  IOSTANDARD LVCMOS18 } [get_ports spi_mosi]                         ; ## D09  FMC_HPC0_LA01_CC_N
+
+set_property  -dict {PACKAGE_PIN  AB3  IOSTANDARD LVDS} [get_ports sysref_out_p]                       ; ## D11  FMC_HPC0_LA05_P
+set_property  -dict {PACKAGE_PIN  AC3  IOSTANDARD LVDS} [get_ports sysref_out_n]                       ; ## D12  FMC_HPC0_LA05_N
+set_property  -dict {PACKAGE_PIN  AA2  IOSTANDARD LVDS} [get_ports rx_sync_p]                          ; ## H10  FMC_HPC0_LA04_P
+set_property  -dict {PACKAGE_PIN  AA1  IOSTANDARD LVDS} [get_ports rx_sync_n]                          ; ## H11  FMC_HPC0_LA04_N
+
+set_property  -dict {PACKAGE_PIN  K20  IOSTANDARD LVCMOS18} [get_ports iic_scl]                        ; ## C30  FMC_HPC0_IIC_SCL
+set_property  -dict {PACKAGE_PIN  L20  IOSTANDARD LVCMOS18} [get_ports iic_sda]                        ; ## C31  FMC_HPC0_IIC_SDA
+set_property  -dict {PACKAGE_PIN  AA7  IOSTANDARD LVDS} [get_ports sysref_p]                           ; ## H4  FMC_HPC0_CLK0_M2C_P
+set_property  -dict {PACKAGE_PIN  AA6  IOSTANDARD LVDS} [get_ports sysref_n]                           ; ## H5  FMC_HPC0_CLK0_M2C_N
+set_property  -dict {PACKAGE_PIN  V2  IOSTANDARD LVCMOS18} [get_ports spi_miso]                        ; ## H7  FMC_HPC0_LA02_P
+set_property  -dict {PACKAGE_PIN  V1  IOSTANDARD LVCMOS18} [get_ports spi_csn_ad9656]                  ; ## H8  FMC_HPC0_LA02_N
+# clocks
+
+#create_clock -name tx_ref_clk     -period  4.00 [get_ports ref_clk0_p]
+create_clock -name rx_ref_clk     -period  4.00 [get_ports ref_clk0_p]
```

Comment:
My understanding is that the clock is running at 125MHz, so it should be constrained appropriately

Reply:
Changed to correct period.